app/components/Dashboard/PriceHistoryPanel/PriceHistoryPanel.jsx

```diff
@@ -101,4 +109,49 @@ export default class PriceHistoryPanel extends React.Component<Props> {
     }
     return date.toLocaleString('en-US', { month: 'numeric', day: 'numeric' })
   }
+
+  renderLatestPrice = () => {
+    const { staticPrice } = this.props
+    return (
+      <div className={styles.currentPrice}>
+        {this.formatPrice(staticPrice, formatFiat)}
+      </div>
+    )
+  }
+
+  formatPrice = (
+    price: number,
+    formatter: Function = formatThousands
+  ): string => {
+    const { symbol } = CURRENCIES[this.props.currency]
```

Comment:
what if there is no symbol for that currency. consider lodash get and some defensive programming around this case